src/generate-packages.ts

```diff
@@ -43,7 +42,7 @@ export default async function generatePackages(dt: FS, allPackages: AllPackages,
         log(` * ${pkg.libraryName}`);
     }
     log("## Generating deprecated packages");
-    withNpmCache(new UncachedNpmInfoClient(), async client => {
+    await withNpmCache(new UncachedNpmInfoClient(), async client => {
```

Comment:
The linter was complaining that there was a unhandled promise here, which appears to be true, but perhaps this was intentional to speed-up the process?